app/models/competition.rb

```diff
@@ -57,6 +57,15 @@ class Competition < ApplicationRecord
            allow_nil: true,
            with_model_currency: :currency_code
 
+  validate :start_date, :cant_change_across_regulation_boundaries, if: lambda { |_lambda|
+    start_date_changed? && start_date_was.present? # start_date_was will be nil when `start_date` changes from `nil` to `true`
```

Comment:
Food for though: I think `start_date_was.present?` implies that `start_date_changed?`. In other words, how can `start_date_changed?` not be true when the value `start_date_was` is `present`?

This suspicion needs to be tried out and validated though. If it turns out to be correct, then checking `start_date_was.present?` is all that is needed.